buildSrc/src/main/java/org/openapitools/codegen/languages/AbstractJavaJAXRSServerCodegen.java

```diff
@@ -49,7 +49,7 @@
     protected boolean useBeanValidation = true;
     protected boolean useTags = false;
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractJavaJAXRSServerCodegen.class);
+    private final Logger LOGGER = LoggerFactory.getLogger(AbstractJavaJAXRSServerCodegen.class);
```

Comment:
We don't want this to be a const?

Reply:
This isn't our code, it's just a local copy of the upstream version, so I think we should try to keep it as close to the original. Here I just merged the changes from the v5.4.0 version of the [file](https://github.com/OpenAPITools/openapi-generator/blob/5.4.x/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaJAXRSServerCodegen.java), while keeping our only [custom change.](https://github.com/candlepin/candlepin/commit/d6a9b9e7fa2b3fcc2bb5ca0cd99423d6f6e6e292#diff-222916468ed7d67decffd7c8eb373abbd37dbd7b6c338821cf1031f29468d927)

Reply:
Ahh, alright